brave/browser-laptop
# Updates recovery dialog text and buttons

Resolves #11857 

## Submitter Checklist:


## Test Plan:
- start browser
- enable payments
- go to recovery dialog
- make sure that it's the same as spec

**New look**
![image](https://user-images.githubusercontent.com/9574457/38814564-0af7fb76-4192-11e8-91d1-1b37257212af.png)


## Reviewer Checklist:


Tests
